swift_browser_ui/upload/cryptupload.py

```diff
@@ -116,23 +154,23 @@ async def add_header(
             self.project, self.container, self.object_name, b64_header
         )
 
-        resp = await self.client.put(
-            common.generate_download_url(
-                self.host,
-                container=self.container,
-                object_name=f"{common.SEGMENTS_PREFIX}{self.object_name}/{self.segment_id}/{0:08d}",
-            ),
-            data=header,
-            headers={
-                "X-Auth-Token": self.token,
-                "Content-Type": "application/swiftclient-segment",
-            },
-            timeout=UPL_TIMEOUT,
-            ssl=ssl_context,
-        )
-        if resp.status == 408:
-            raise aiohttp.web.HTTPRequestTimeout()
-        LOGGER.debug(f"Successfully uploaded header for {self.object_name}.")
+        # resp = await self.client.put(
```

Comment:
Are these commented lines leftovers?

Reply:
Yep, uploading the header into Allas causes issues with the download logic, so I deprecated this part – I'll have to remove that, thanks for the reminder 👍🏼